run_tests.py

```diff
@@ -2,8 +2,6 @@
 # -*- coding: utf-8 -*-
 """Script to run the tests."""
 
-from __future__ import print_function
```

Comment:
this is needed if someone runs setup.py with python2

Reply:
Python 2 is dead. You only support Python >= 3.7

Reply:
correct python 2 is no longer maintained/supported, but that does not mean someone is still able to run it

Reply:
https://github.com/log2timeline/dfdatetime/blob/de021f3d198c47bd6d616dc68bd59f83a6158961/setup.py#L25-L28

Reply:
It will fail at the f-string.

Reply:
Good point I'll have a look at a later point at this.